src/lib/src/spinner/overlay.spinner.component.css

```diff
@@ -0,0 +1,10 @@
+.overlay {
+  position: fixed;
```

Comment:
Would there be some way to also have this module support absolute-positioning?  (Maybe you could switch between whole-page and within-relatively-positioned-element with an `@Input` of some sort?)

I was imagining (at least for metrics) that this would just overlay the graph, not the whole page.  But maybe others were headed in the whole page direction ... so seems prudent to support both here!

Reply:
I understood the request to be also cover the controls with the spinner overlay, which is the whole page because there's a control in the header too.

Reply:
I mean, my personal feeling is that overlaying the whole page is too heavy handed.  (You can't click/navigate anything... you're stuck waiting).  But I guess I'd have to actually try it first.

I was thinking more overlay the graph, and disable/show-spinner on the TBD `GO ->` button.  The dropdown in the header feels like more of a navigation item, not a facet of the graph, so I'd leave it enabled.  So you're allowed to navigate away, but not change facets mid-calculation.

But I have all the feels.  So yeah, I think it's useful to just add both to the styleguide and try some stuff.

Reply:
Or it could be equally as easy to just add a 3rd spinner-y component to handle this, and not abuse your current global-overlay.